spec/rails_helper.rb

```diff
@@ -0,0 +1,62 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
+require 'spec_helper'
+
+require File.expand_path('../dummy/config/environment', __FILE__)
+
+require 'rspec/rails'
+require 'capybara/rails'
+require 'phantomjs/poltergeist'
+require 'database_cleaner'
+
+Capybara.javascript_driver = :poltergeist
+
+def reload_menus!
+  ActiveAdmin.application.namespaces.each(&:reset_menu!)
+end
+
+def reload_routes!
+  Rails.application.reload_routes!
+end
+
+# Setup ActiveAdmin
+ActiveAdmin.application.load_paths = [File.expand_path('../dummy/app/admin', __FILE__)]
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.